manifests/init.pp

```diff
@@ -109,13 +109,14 @@
         false => Anchor['pgbouncer::begin'],
       }
       package{ $pgbouncer_package_name:
-        ensure  => installed,
-        require => $package_require,
+        ensure => installed,
+        before => $package_require,
```

Comment:
This reverses the ordering. On a RedHat box, it would say install package _before_ setting up the repo, which is obviously a bad thing to do.